webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelect.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import TypeAheadSelect from '.';
+import { initialState, props } from './TypeAheadSelect.fixtures';
+import Story from '../../../../../..//stories/components/Story';
+
+const mockStore = configureMockStore([thunk]);
+
+export default {
+  title: 'Components|TypeAheadSelect',
+};
+
+export const typeAhead = () => (
+  <Story>
+    <TypeAheadSelect store={mockStore(initialState)} {...props} />
```

Comment:
This is failing: `Invariant Violation: could not find react-redux context value; please ensure the component is wrapped in a <Provider>
`

I tried using a Provider but that doesn't seem to work either. In fact, I see the same problem in FactChatModal stories

Reply:
@jeremylenz I see you were involved in some of the code around Storybook lately. Any thoughts on this?

Reply:
@jturel I think you just need to add the storeDecorator; you can find examples in some other `.stories` files.